README.md

```diff
@@ -32,35 +32,30 @@ for documentation build:
 
 *   doxygen http://www.stack.nl/~dimitri/doxygen/
 
+for rpm building:
+
+*   tito https://github.com/dgoodwin/tito
```

Comment:
Please add build command example

Reply:
Sorry , please rephrase that. From first look I would understand that as a command to build. Build example command is bellow and it is fine.